test/zod-utils.test.ts

```diff
@@ -46,6 +93,19 @@ describe('zodDeepPartial', () => {
     expect(deepPartial.parse({})).toEqual({})
 
     expect(() => deepPartial.parse({ n: 'string' })).toThrow()
+
```

Comment:
I added this test and it passes. This works because `zodDeepPartial` is recursive: https://github.com/cau777/ts-mongo/blob/a6011a6b376c276db4c0d0b32bb81871da79e1e4/src/zod-utils.ts#L97-L97

Reply:
Thanks!